mixer/cmd/bundles.go

```diff
@@ -155,31 +155,19 @@ var bundleListCmd = &cobra.Command{
 
 // Bundle Edit command ('mixer bundle edit')
 type bundleEditCmdFlags struct {
-	copyOnly bool
-	add      bool
-	git      bool
+	add bool
+	git bool
 }
 
 var bundleEditFlags bundleEditCmdFlags
 
 var bundleEditCmd = &cobra.Command{
 	Use:   "edit <bundle> [<bundle>...]",
-	Short: "Edit local and upstream bundles, or create new bundles",
-	Long: `Edits local and upstream bundle definition files. This command will locate the
-bundle (looking first in local-bundles, then in upstream-bundles), and launch
-an editor to edit it. If the bundle is only found upstream, the bundle file will
-first be copied to your local-bundles directory for editing. If the bundle is
-not found anywhere, a blank template is created with the correct name. When the
-editor closes, the bundle file is then parsed for validity.
-
-The editor is configured via environment variables. VISUAL takes precedence to
-EDITOR. If neither are set, the tool defaults to nano. If nano is not installed,
-the tool will skip editing, and act as if '--suppress-editor' had been passed.
-
-Passing '--suppress-editor' will suppress launching the editor, and will thus
-only copy the bundle file to local-bundles (if it is only found upstream), or
-create the blank template (if it was not found anywhere). This can be useful if
-you want to add a bundle to local-bundles, but wish to edit it at a later time.
+	Short: "Create new bundles or copy existing bundles",
+	Long: `Create new bundles or copy existing bundles. This command will locate the
+bundle (looking first in local-bundles, then in upstream-bundles). If the bundle is only found upstream,
```

Comment:
> This command will locate the bundle (looking first in local-bundles, then in upstream-bundles).
Sounds better written as:

`This command will locate the bundle by first looking in local-bundles, and then in upstream-bundles.`